tests/app/celery/test_scheduled_tasks.py

```diff
@@ -519,16 +519,16 @@ def test_check_for_missing_rows_in_completed_jobs_uses_sender_id(mocker, sample_
         [MockServicesWithHighFailureRate("123", 0.3)],
```

Comment:
I think the reason this error wasn't caught in your unit tests is because your tests don't match production data close enough. Here I think we should be setting the first argument of `MockServicesWithHighFailureRate` with a UUID as opposed to a string